docs/docs/guides/with-expo.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 4
```

Comment:
I think this position is incorrect (looks like there are several with position 4) but it should probably come after the others but before the Amazon fire support one.